tests/DuskTestCase.php

```diff
@@ -41,4 +42,42 @@ protected function driver()
             )
         );
     }
+
+    public $passwordDefault = 'lifull@123';
+
+    /**
+     * Create user can see blog.
+     *
+     * @return $user
```

Comment:
Can we know return type of $user?
Ex: @return array(or other type) $user
If possible, modify for userCannotSee() and userCanSeeAndDelete() too.

Reply:
I fixed